submissions/Semka094/hooli-style-popup/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap"
+          rel="stylesheet">
+    <title>Pop-up</title>
+</head>
+
+<body>
+
+<div class="menuContainer">
+    <nav class="navWrapper">
```

Comment:
If you have a nested navigation use a nested list of links:
```HTML
    <nav>
        <ul>
            <li><a href="#">Link-1</a></li>
            <li><a href="#">Link-2</a></li>
            <li><a href="#">Link-3</a></li>
            <li><ul>
                <li><a href="#"></a>Link-1</li>
                <li><a href="#"></a>Link-2</li>
                <li><a href="#"></a>Link-3</li>
                <li><a href="#"></a>Link-4</li>
            </ul></li>
            <li><a href="#">Link-5</a></li>
        </ul>
    </nav>
```

Reply:
I have added nested navigation.